Added tiny amount of docks to solve my particula "problem". (#137)
rust-db/refinery: refinery_cli/README.md

refinery migrate -c sqlite_refinery.toml files -p ./sql_migrations
```

> ### Alternativ for docker

Reviewer: ```suggestion
### Running via database uri
```

> To run migrations given that you have a database [uri](http://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-CONNSTRING) (like: postgres://user_name:passwd@hostname:5432/myDB ) stored in an environment variable DB_URI.

Reviewer: ```suggestion
To run migrations from a database [uri](http://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-CONNSTRING) (like: postgres://user_name:passwd@hostname:5432/myDB ) stored in an environment variable DB_URI.
```

```sh
refinery migrate -e DB_URI files -p ./sql_migrations
```

> This option is extra useful when running refinery inside a docker container, were you usualy have the db connection info stored as an environment variable.  

Reviewer: ```suggestion
This option is also useful when running refinery inside a docker container, where you usually have the db connection info stored as an environment variable.  
```
